tle/cogs/codeforces.py

```diff
@@ -61,6 +71,7 @@ def __init__(self, bot):
         delta  | -300 | -200 | -100 |  0  | +100 | +200 | +300
         points |   2  |   3  |   5  |  8  |  12  |  17  |  23
         """
+        await self._validate_gitgud_status(ctx,delta=None)
```

Comment:
Now that I think more about it, its probably better to move this down where the challenge is selected both to avoid the None case and to allow users to see offered upsolves even when they have an active challenge. Sorry for not considering that first time around.

Other than that, the PR is good to merge.

Reply:
Hmm .. If you don't mind, I'd like to discuss that.
From the functionality perspective, I think that one of `gitgud` goals is to let the user be challenged to work on one problem for a specific duration (3 hours iirc). So I think that the user should be disallowed to check another problems while working on this problem.